static/usage/item/helper-error/react.md

```diff
@@ -0,0 +1,41 @@
+```tsx
+import React, { useState } from 'react';
+import { IonInput, IonItem, IonLabel, IonNote } from '@ionic/react';
+
+function Example() {
+  const [isTouched, setIsTouched] = useState(false);
+  const [isValid, setIsValid] = useState(false);
+  const [isInvalid, setIsInvalid] = useState(false);
```

Comment:
Since the input can't be valid and invalid at the same time, you could do this with just `isValid` and add the `ion-invalid` class if it's false.